packages/vue/src/components/SpTokenTransfer/SpTokenTransfer.vue

```diff
@@ -399,7 +400,6 @@ export default defineComponent({
       return amount == '' ? 0 : parseInt(amount)
     }
     let resetTx = (): void => {
-      state.tx.amount = []
```

Comment:
This not only does not solve the issue but it breaks `resetTx` for it won't reset everything anymore.

What you need is to call `bootstrapTxAmount` after tx is done. But then, watch out for timing issues.